dev/source/docs/using-linux-trace-toolkit-ng-lttng-to-trace-ardupilot-in-realtime.rst

```diff
@@ -40,11 +40,11 @@ Monitoring APM with LTTng
 
 In order to monitor APM with *LTTng*, the first thing to do is to enable
 support for *LTTng* in ArduPilot builds. Using the regular make build
-system, you do this by modifying `mk/board_native.mk line 30 <https://github.com/ArduPilot/ardupilot/blob/master/mk/board_native.mk#L30>`__:
+system, you do this by modifying `mk/board_native.mk line 38 <https://github.com/ArduPilot/ardupilot/blob/master/mk/board_native.mk#L38>`__:
```

Comment:
having to change this line to compile is awful. Can we just direct people to the waf build system?